docs/pages/access-controls/guides/per-session-mfa.mdx

```diff
@@ -9,6 +9,7 @@ when starting new:
 
 - SSH connections (a single `tsh` call)
 - Kubernetes sessions (a single `kubectl` call)
+- Database sessions (a single `tsh db connect` call)
```

Comment:
Should we also clarify "a single tsh db connect call or a database connection established through the local proxy tsh db proxy"?

Reply:
Sure, I'll add a note to the Admonition about DB access below.